api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -87,14 +90,20 @@
   @Autowired private SurveyModuleDao surveyModuleDao;
   @Autowired private JdbcTemplate jdbcTemplate;
   @Autowired private CohortBuilderMapper cohortBuilderMapper;
+  @Autowired private Provider<WorkbenchConfig> configProvider;
   @Mock private WorkspaceAuthService workspaceAuthService;
-  @Mock private Provider<WorkbenchConfig> configProvider;
   @Mock private Provider<MySQLStopWords> mySQLStopWordsProvider;
 
   @TestConfiguration
   @Import({FakeClockConfiguration.class, CohortBuilderMapperImpl.class})
   @MockBean({WorkspaceAuthService.class})
-  static class Configuration {}
+  static class Configuration {
+    @Bean
+    @Scope("prototype")
+    WorkbenchConfig workbenchConfig() {
+      return workbenchConfig;
+    }
+  }
```

Comment:
Why are we adding this to test? It's not being used.

Reply:
Added to enable testing ```  @Test
  public void testUniversalSearch_feature() {
    workbenchConfig.featureFlags.enableUniversalSearch = false;
    assertThat(
        controller
            .isEnabledUniversalSearch()
            .getStatusCode())
        .isEqualTo(HttpStatus.NOT_IMPLEMENTED);
    workbenchConfig.featureFlags.enableUniversalSearch = true;
    assertThat(
        controller
            .isEnabledUniversalSearch()
            .getStatusCode())
        .isEqualTo(HttpStatus.OK);
  }```
Will remove these since the test is no longer there...